ggml.c

```diff
@@ -100,6 +100,45 @@ typedef void * thread_ret_t;
 #include <hbwmalloc.h>
 #endif
 
+#if (defined(__linux__) || defined(__APPLE__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)) && \
+    (!defined(TARGET_OS_TV))
+
```

Comment:
Does anyone know how to disable this branch when building on tvOS ?

![image](https://github.com/ggerganov/llama.cpp/assets/1991296/5514c551-bcd7-44bd-9b21-ad8f86a5c9e8)

Reply:
Might need to include `TargetConditionals.h` for `TARGET_OS_TV` to be available.